### PR TITLE
Update README.md with Meta and References

### PR DESCRIPTION
**Pull request by jeffcpullen**
Did not include internal RH links since those would be inaccessible by most.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ Basics
 * New roles should be initiated in line with the skeleton directory, which has standard boilerplate
   code for a Galaxy-compatible Ansible role and some enforcement around these standards
 
+Meta Best Practices
+===================
+
+A rationale should be included for best practices, with a reference if applicable. It is really helpful to know not only how to do certain things, but why to do them in this way. It will also help with further revisions of the standards as some items may become obsolete or no longer applicable.  If the reason is not included, there is a risk of keeping items that are no longer applicable, or alternatively blindly removing items that should be kept. It also has great educational value for understanding how things actually work (or how they don’t).
+
```
Comment (pcahyna) [inline README.md:16]: I would move it above, just below the introductory sentence. It is even more basic than the "Basics".

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ Basics
 * New roles should be initiated in line with the skeleton directory, which has standard boilerplate
   code for a Galaxy-compatible Ansible role and some enforcement around these standards
 
+Meta Best Practices
+===================
+
+A rationale should be included for best practices, with a reference if applicable. It is really helpful to know not only how to do certain things, but why to do them in this way. It will also help with further revisions of the standards as some items may become obsolete or no longer applicable.  If the reason is not included, there is a risk of keeping items that are no longer applicable, or alternatively blindly removing items that should be kept. It also has great educational value for understanding how things actually work (or how they don’t).
```
Comment (greg-hellings) [inline README.md:15]: Please give some line breaks in this paragraph around the 80-90 character width to facilitate reading and editing in code editors like vim.

**Review by greg-hellings — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -111,3 +116,14 @@ Vars vs Defaults
...
   which should default to an empty array in `defaults/main.yml` and be documented as such.
+  
+References
+==========
+
+Links that contain additional standardization information that provide context, inspiration or contrast to the standards described above.
+
+* << Placeholder for RH internal links >>
+* https://github.com/debops/debops/blob/v0.7.2/docs/debops-policy/code-standards-policy.rst). For inspiration, as the DebOps project has some specific guidance that we do not necessarily want to follow.
+* https://github.com/oasis-roles/meta_standards
```
Comment (pcahyna) [inline README.md:127]: this is too meta

**Comment by pcahyna** [comment]
Can be closed, in favor of #15

**Pull request closed without merging**